jwst/lib/tests/test_set_telescope_pointing.py

```diff
@@ -52,6 +52,83 @@
 # ########################
 
 
+@pytest.fixture(scope='module')
+def method_gscmd_j3pags():
+    """Calculate matricies using the GSCMD_J3PAGS method
+
+    This set was derived from the first valid group of enginerring parameters for exposure
+    jw00624028002_02101_00001_nrca1 retrieved from the SDP regression tests for Build 7.1.1.
+    """
+    # setup inputs
+    t_pars = stp.TransformParameters()
+
+    t_pars.guide_star_wcs = stp.WCSRef(ra=241.24294932221, dec=70.66165389073196, pa=None)
+    t_pars.pointing = stp.Pointing(
+        q=np.array([-0.20954692, -0.6177655, -0.44653177, 0.61242575]),
+        j2fgs_matrix=np.array([-9.77300013e-04, 3.38988895e-03, 9.99993777e-01,
+                               9.99999522e-01, 8.37175385e-09, 9.77305600e-04,
+                               3.30458575e-06, 9.99994254e-01, -3.38988734e-03]),
+        fsmcorr=np.array([0.00584114, -0.00432878]),
+        obstime=Time(1611628160.325, format='unix'),
+        gs_commanded=np.array([-22.40031242,  -8.17869377])
+    )
+    t_pars.siaf = stp.SIAF(v2_ref=120.525464, v3_ref=-527.543132, v3yangle=-0.5627898, vparity=-1,
+                           crpix1=1024.5, crpix2=1024.5, cdelt1=0.03113928, cdelt2=0.03132232,
```

Comment:
Aren't CDELT's in units of degrees, in which case these values represent **really** big pixels.

Reply:
Or are they in units of arcsec in the SIAF tables?

Reply:
This was pulled from the SIAF, so presumption is that the values are "correct". Sure does look like arcsec.